llvmlite/binding/targets.py

```diff
@@ -192,23 +192,11 @@ def get_element_offset(self, ty, position):
                              "type?".format(position, str(ty)))
         return offset
 
-    def get_pointee_abi_size(self, ty):
```

Comment:
Is this removal a breaking change? I see Numba doesn't use it, but it is documented: https://llvmlite.readthedocs.io/en/latest/user-guide/binding/target-information.html#llvmlite.binding.TargetData.get_pointee_abi_size

Can / should this be kept, but only usable when typed pointers are enabled, then removed later when typed pointer support is removed?

Reply:
That's a good point, maybe it should be kept. If we keep it, it will definitely only work in typed pointers mode though.

Most use cases for it I can think of are covered with `get_abi_size` and `get_abi_alignment`, so I don't think it's vitally important, but I'll have a look and if we can keep it in typed pointers mode, we might as well.

Reply:
I've put this back with a runtime error if opaque pointers are enabled. This will likely have to be removed once we upgrade to the next versions of LLVM though.